modules/tinymce/src/core/test/ts/browser/paste/ImagePasteTest.ts

```diff
@@ -5,23 +5,32 @@ import { TinyAssertions, TinyDom, TinyHooks, TinySelections } from '@ephox/wrap-
 import { assert } from 'chai';
 
 import Editor from 'tinymce/core/api/Editor';
+import { EditorEvent } from 'tinymce/core/api/util/EventDispatcher';
 import * as Clipboard from 'tinymce/core/paste/Clipboard';
 
 describe('browser.tinymce.core.paste.ImagePasteTest', () => {
+  let lastInputEvent: EditorEvent<InputEvent> | undefined;
```

Comment:
Use a `Singleton.value` wraps cell and option.

Reply:
Good to know, will use `Singleton.value` in similar situations in the future.